src/components/molecules/NavBar/NavBar.scss

```diff
@@ -28,6 +28,11 @@ $border-radius: 28px;
     cursor: pointer;
   }
 
+  .navbar-profile-image-container {
+    width: 40px;
+    height: 40px;
+  }
```

Comment:
since we have `--large`, can we remove these values?

Reply:
I changed 50x50 to 40x40 to implement it as you suggested

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/5bcc570daf621979becff60da99f34b765b09c03